src/fs/manifest/schema.rs

```diff
@@ -294,24 +360,59 @@ impl CloudflareConfig {
         let worker_name = args.cloudflare_worker_name().map(ToString::to_string).or_else(|| self.worker_name.clone())
             .or(wranger_worker_name)
             .ok_or_else(
-                || miette!("No Cloudflare worker name provided. You must provide the name of a Cloudflare worker to deploy to.")
+                || miette!("No Cloudflare worker name provided. You must provide the name of a Cloudflare worker, either via an environment variable, a CLI flag, or in your MultiTool.toml file or Wrangler.toml file.")
```

Comment:
Not necessarily a `toml` file. I'd be down to use `manifest file` here but we'll want to make sure we document the manifest file in `Cathy.`

Reply:
I pinged Anita about documenting the manifest file (as a page in Cathy),